run-your-c++-code-on-the-web/README.md

```diff
@@ -1,3 +1,7 @@
+_By [Stefan Verhoeven](https://orcid.org/0000-0002-5821-2060), [Faruk Diblen](https://orcid.org/0000-0002-0989-929X),
+[Jurriaan H. Spaaks](https://orcid.org/0000-0002-7064-4069), [Adam Belloum](https://orcid.org/0000-0001-6306-6937), and
+[Christiaan Meijer](https://orcid.org/0000-0002-5529-5761)._
```

Comment:
Nice to use orcid ids, better than author link to medium acount, GitHub account or NLeSC employee page.

In https://github.com/NLESC-JCER/run-cpp-on-web/issues/34#issuecomment-691934446 I added the team and NLeSC, is there a reason why you don't mention the team?

Reply:
I honestly don't think the team is an interesting entity to the outside world. Including any information that is not for the reader, but rather for colleagues or bosses, sends a message that whoever wrote the article cares more about pushing their information than helping the reader with their needs.

Reply:
OK, that makes sense.